lib/components/gene/__tests__/__snapshots__/about-test.js.snap

```diff
@@ -0,0 +1,197 @@
+exports[`test renders correctly 1`] = `
+<View>
+  <View>
+    <View
+      style={
+        Object {
+          "marginLeft": 0,
+          "marginRight": 0,
+        }
+      }>
+      <Text
+        accessible={true}
+        allowFontScaling={true}
+        ellipsizeMode="tail"
+        numberOfLines={0}
+        style={
+          Array [
+            Object {
+              "fontSize": 17,
+            },
+            Object {
+              "fontSize": 16,
+              "lineHeight": 20,
+              "marginBottom": 20,
+            },
+            Object {
+              "fontFamily": "AGaramondPro-Regular",
+            },
+          ]
+        }>
+        This is some example text
+      </Text>
```

Comment:
so, the image you've got here is with actual content though right?

What you've got in this file, is a very minimal version of it.

Also, is the idea that, for the test, you write out the view xml? or was this partially generated or something?